android/fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -0,0 +1 @@
+Safely manage your Wit coins with myWitWallet
```

Comment:
Is there a way to provide this information in a unique configuration file instead of having multiple files with one line?